src/CriticalTransitions.jl

```diff
@@ -16,6 +16,11 @@ using Symbolics
 using Optim, Dierckx
 using Printf, DrWatson, Dates, Statistics
 
+# Define custom types
+Parameters = Union{Vector{Any}, Nothing};
```

Comment:
I agree, I just re-added these custom types because some functions are still dependent on them, which caused an error when they were removed. In the long-run we should follow the style conventions, absolutely.